src/app/modules/auth/sign-up/sign-up.component.html

```diff
@@ -14,120 +13,222 @@
+            <div class="input-box required-field">
+              <label for="efname">{{ 'Email' | translate }}</label>
+              <div class="input-box__border">
+                <input
+                  placeholder="Email"
+                  type="email"
+                  id="email"
+                  #email="ngModel"
+                  name="email"
+                  [(ngModel)]="signupObject.email"
+                  required
+                  email
+                />
               </div>
+              <div
+                *ngIf="email.invalid && (email.touched || signupForm.submitted)"
+                class="error-msg"
+              >
+                <div *ngIf="email.errors.required">
+                  {{ 'Email is Required' | translate }}
+                </div>
+                <div *ngIf="email.errors.email">
+                  {{ 'Email must be a valid email address' | translate }}
+                </div>
+              </div>
+            </div>
 
-              <div class="input-checbox">
-                  <input class="cm-checkbox" type="checkbox" #agreedForTermsCondition="ngModel" id="agreedForTermsCondition"  name="agreedForTermsCondition" [(ngModel)]="signupObject.agreedForTermsCondition" required />
-                  <label for="agreedForTermsCondition">
-                    {{"I'm good with the Civis"| translate}}<br>
-                    <a href="/terms" class="policy-link">{{'Terms of Service,' | translate}}</a>
-                    &nbsp;
-                    <a href="/privacy" class="policy-link">{{'Privacy Policy' | translate}}</a>
-                    &
-                    <a href="/content-policy" class="policy-link">{{'Content Policy.' | translate}}</a>
-                  </label>
+            <div class="input-box required-field">
+              <label for="password">{{ 'Password' | translate }}</label>
+              <div class="input-box__border">
+                <input
+                  placeholder="At least 8 characters"
+                  type="password"
+                  id="password"
+                  #password="ngModel"
+                  name="password"
+                  pattern='(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[$@$!%*?&])[A-Za-z\d$@$!%*?&].{8,}'
+                  [(ngModel)]="signupObject.password"
+                  required
+                />
               </div>
-              <div class="error-msg"  *ngIf="agreedForTermsCondition.invalid && (agreedForTermsCondition.touched || signupForm.submitted)">
-                {{'Please accept our Terms of Service & Privacy Policy to continue.' | translate}}
+              <div
+                *ngIf="
+                  password.invalid && (password.touched || signupForm.submitted)
+                "
+                class="error-msg"
+              >
+                {{ signupObject.password ? 'Password is invalid' : ('Password is Required' | translate) }}
```

Comment:
@Harshwardhan18 Let the user again know the password requirements here, just saying invalid wouldn't help the user